### PR TITLE
Move circle to redis 6-rc3

### PR DESCRIPTION
**Pull request by ashtul**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,8 +7,8 @@ ARG OSNICK=buster
 ARG ARCH=x64
 
 #----------------------------------------------------------------------------------------------
-# FROM redis:latest AS builder
-FROM redisfab/redis-${ARCH}-${OSNICK}:5.0.5 AS builder
+FROM redis:6.0-rc3 AS builder
```
Comment (danni-m) [inline Dockerfile:10]: @ashtul @rafie won't this break ARM build?

Comment (rafie) [reply Dockerfile:10]: Should be `FROM redisfab/redis:6.0-rc3-${ARCH}-bionic AS builder`.

File: Dockerfile [hunk 2]
```diff
@@ -22,8 +22,8 @@ RUN echo nproc=$(nproc); echo NPROC=$(eval "$X_NPROC")
 RUN make build
 
 #----------------------------------------------------------------------------------------------
-# FROM redis:latest
-FROM redisfab/redis-${ARCH}-${OSNICK}:5.0.5
+FROM redis:6.0-rc3
```
Comment (rafie) [inline Dockerfile:25]: Should be `FROM redisfab/redis:6.0-rc3-${ARCH}-bionic`

File: Dockerfile (hunk 1 above)
Comment (ashtul) [reply Dockerfile]: Isn't the arm builder in file `Dockerfile.arm`?
For now, left as is since there is no `-xbuild` version.

Comment (rafie) [reply Dockerfile]: Well, it's a little bit funny. The Dockerfile.arm will build you an arm version when you're running it on an intel hardware, which is kind of cool. But what if you're running on arm hardware and want to build natively? Then you use Dockerfile, and the ARCH arg tells it which redis image it should get. Really, such a pity nobody uses it. :-)

**Comment by danni-m** [comment]
@rafie is it OK now?

File: Dockerfile [hunk 3]
```diff
@@ -8,7 +8,8 @@ ARG ARCH=x64
 
 #----------------------------------------------------------------------------------------------
 # FROM redis:latest AS builder
-FROM redisfab/redis-${ARCH}-${OSNICK}:5.0.5 AS builder
+# FROM redisfab/redis-${ARCH}-${OSNICK}:5.0.5 AS builder
```
Comment (rafie) [inline Dockerfile:11]: can remove comment.

File: Dockerfile [hunk 4]
```diff
@@ -23,7 +24,8 @@ RUN make build
 
 #----------------------------------------------------------------------------------------------
 # FROM redis:latest
-FROM redisfab/redis-${ARCH}-${OSNICK}:5.0.5
+# FROM redisfab/redis-${ARCH}-${OSNICK}:5.0.5
```
Comment (rafie) [inline Dockerfile:27]: can remove comment.

**Pull request merged**